### ISSUE TITLE
Automatically disable kube-ps1 when no context is available

### ISSUE THREAD

**Issue by dashie** [jonmosco/kube-ps1#179]
Is it possibile to automatically disable kube-ps1 when no context is set?
So have a empty string instead of `(<symbol>|N/A:N/A)`

**Comment by credativ-dar** [comment]
Just in case you missed that: `kubeoff -g` turns it off by default so you can just `kubeon` if you need it, I guess that's easier for most people than unsetting the context.

**Comment by jonmosco** [comment]
Yes, that is the solution for this is using the `kubeon` and `kubeoff` functions: https://github.com/jonmosco/kube-ps1?tab=readme-ov-file#enablingdisabling

**Comment by gitmpr** [comment]
kubeon and kubeoff only remove the kube-ps1 part from the PS1 string for the current shell. This is not exactly the requested feature here.

What we want instead is to conditionally remove the kube-ps1 part from the PS1 if no context is set. This can be achieved easily by adding an extra check and return early if no context is set

```bash
[[ -z "${KUBE_PS1_CONTEXT}" ]] && [[ "${KUBE_PS1_CONTEXT_ENABLE}" == true ]] && return
```

[at the start of the  `kube_ps1()` function](https://github.com/jonmosco/kube-ps1/blob/b2286b63edf513ec0bfc425b55ceaffcf7e37b48/kube-ps1.sh#L378)

@jonmosco 
If you can acknowledge this I can make a PR and also make it configurable with a KUBE_PS1_HIDE_IF_NOCONTEXT environment variable setting, set to false by default

**Comment by jonmosco** [comment]
https://github.com/jonmosco/kube-ps1/pull/198
